### PR TITLE
add Reloader annotations to trino-coordinator deployment in osc-cl2

### PR DESCRIPTION
**Pull request by dystewart**
A follow up to [https://github.com/operate-first/apps/issues/1778](https://github.com/operate-first/apps/issues/1778)

### CODE REVIEW

File: trino/base/trino-coordinator-dc.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  annotations:
+    configmap.reloader.stakater.com/reload: "trino-config-volume, trino-catalogs-volume"
```
[the remark this answers is not in the document]
Comment (HumairAK) [reply trino/base/trino-coordinator-dc.yaml:5]: these are the configmaps we manage for osc-cl2 trino via argocd: https://github.com/operate-first/apps/blob/master/kfdefs/overlays/osc/osc-cl2/trino/configs/kustomization.yaml

File: trino/base/trino-coordinator-dc.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+    secret.reloader.stakater.com/reload: "s3buckets, trino-config-monitoring, trino-oauth"
```
Comment (HumairAK) [inline trino/base/trino-coordinator-dc.yaml:6]: remove `trino-config-monitoring`

**Comment by HumairAK** [comment]
@dystewart can you look at the worker deployment here: https://github.com/operate-first/odh-manifests/blob/osc-cl1-v1.1.2/trino/base/trino-worker-dc.yaml 

and also add the relevant annotations for the secrets/configmaps being consumed here. 
looks to me like for configmaps we want: trino-configfiles,  trino-catalog 
and for secrets we want: s3buckets

**Comment by dystewart** [comment]
@HumairAK Added annotations to trino worker deployment.

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**